src/controlnet_aux/open_pose/__init__.py

```diff
@@ -196,11 +196,14 @@ def detect_poses(self, oriImg, include_hand=False, include_face=False) -> List[P
             
             return results
         
-    def __call__(self, input_image, detect_resolution=512, image_resolution=512, include_body=True, include_hand=False, include_face=False, hand_and_face=False, return_pil=True):
-        if hand_and_face:
+    def __call__(self, input_image, detect_resolution=512, image_resolution=512, include_body=True, include_hand=False, include_face=False, hand_and_face=None, return_pil=None, output_type="pil"):
```

Comment:
Would do the same changes as in https://github.com/patrickvonplaten/controlnet_aux/pull/49/files#r1214565652

To deprecate a function argument we must remove it from the signature (can't have both `return_pil` and `output_type=="pil"` in the signature at once - too confusin gto read